src/com/vamshi/javaprogramplementation.java

```diff
@@ -0,0 +1,12 @@
+package com.vamshi;
+
+public class javaprogramplementation extends Javaprog{
+
+	public static void main(String[] args) {
+		javaprogramplementation j = new javaprogramplementation();
+		System.out.println(j.averagegrade());
+        System.out.println("CheckLeapYear"+' '+j.CheckLeapYear());
+        System.out.println(j.factorial());
```

Comment:
once you have object for Javaprog
you can call methods for those classes using that object